app/src/main/java/org/schabi/newpipe/util/StreamDialogEntry.java

```diff
@@ -60,20 +62,31 @@
      * Info: Add this entry within showStreamDialog.
      */
     enqueue(R.string.enqueue_stream, (fragment, item) -> {
-        NavigationHelper.enqueueOnPlayer(fragment.getContext(), new SinglePlayQueue(item));
+        fetchItemInfoIfSparse(fragment, item,
+                fullItem -> NavigationHelper.enqueueOnPlayer(fragment.getContext(), fullItem)
```

Comment:
I think all lambda parameters should lie on the same line (also below) ;-)
```suggestion
        fetchItemInfoIfSparse(fragment, item, fullItem ->
                NavigationHelper.enqueueOnPlayer(fragment.getContext(), fullItem)
```